packages/api/src/apis/collectionsApi/endpoints/getIndividualGroups/e2e.spec.js

```diff
@@ -60,7 +60,7 @@ apiDescribe('individualGroup', () => {
       })
     })
     describe('by invalid catalogNumber', () => {
-      it('Succeed with invalid catalogNumber', () => {
+      it('Return empty array with invalid catalogNumber', () => {
```

Comment:
this is picky and I know that a lot of the current tests do not follow this convention, but ideally the `it` statement should be possible to read as a grammatically correct sentence. so in this case it would be "it returns..." instead of "it Return".